app/src/main/java/com/uncookthebook/app/ReportArticleFragment.java

```diff
@@ -73,8 +74,40 @@ public void onStart() {
 
     @Override
     public void onDestroy() {
-        super.onDestroy();
         APIServiceUtils.getOkHttpClient().dispatcher().cancelAll();
+        sendSubmitReport();
+        super.onDestroy();
+    }
+
+    private void sendSubmitReport(){
+        //check if one of the two buttons has been clicked
+        boolean legitClicked = animationHandler.hasViewBeenClicked(view.findViewById(R.id.button_legit));
+        boolean fakeCliked = animationHandler.hasViewBeenClicked(view.findViewById(R.id.button_fake));
+
+        if(legitClicked || fakeCliked){
+            ReportValue reportValue = legitClicked ? ReportValue.LEGIT : ReportValue.FAKE;
+            sendReportValue(reportValue);
+        }
+    }
+
+    private void sendReportValue(ReportValue reportValue){
+        APIService apiService = APIServiceUtils.getAPIServiceClient();
+        GoogleSignInAccount account = ((GoogleActivity) Objects.requireNonNull(getActivity())).getGoogleAccount();
+        apiService.submitReport(new TokenizedRequest<>(account.getIdToken(), new SubmitReportRequest(url.toString(), reportValue))).enqueue(new Callback<Void>() {
+            @Override
+            public void onResponse(Call<Void> call, Response<Void> response) {
+                if(response.code() == 201){
+                    Log.d(TAG, "Submit report successful");
+                }else{
+                    Log.w(TAG, "Submit report failed");
```

Comment:
I think we should do something in case the call fails. Its not good if the user thinks he submitted but in reality he didn't. We have to discuss this